### PR TITLE
Add vista3d

### LINKED ISSUE #604: Add Vista3d Bundle
Author: @heyufan1995 @wyli

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes #604 .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/vista3d/docs/data_license.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Third Party Licenses
+-----------------------------------------------------------------------
+
+/*********************************************************************/
+i. LUng Nodule Analysis 2016
```
Comment (yiheng-wang-nv) [inline models/vista3d/docs/data_license.txt:5]: Hi @heyufan1995 @wyli , do you think we should use this data license? The Lung dataset seems not mentioned in the bundle

**Review by KumoLiu — APPROVED** [review]

**Pull request merged**